maquetado/scss/partials/_tech.scss

```diff
@@ -51,3 +46,37 @@ $border-radius: 2%;
   font-weight: bold;
   margin: 0 0 $single-margin * 2 0;
 }
+
+@media screen and (max-width: $breakpoint) {
+
+  .tech-box {
+    position: relative;
+    width: 80%;
+
+    &::after {
+      background-color: $color-green;
+      content: '';
+      height: 6px;
+      transform: translate(0,-200px);
+      position: absolute;
+      width: 100%;
+    }
+  }
+  
+  .tech-box-feature {
+    justify-self: center;
+    grid-row: auto;
+    &::after {
+      width: 0;
+    }
+  } 
+  
+  .tech-img {
+    height: $tech-logo-size * 3;
```

Comment:
Please check out how this looks on mobile. It looks kinda weird to me. 

<img width="898" alt="bildschirmfoto 2018-11-02 um 3 18 12 pm" src="https://user-images.githubusercontent.com/3473831/47938583-97f8a580-deb2-11e8-98b1-16dda5b925a5.png">